src/frontend/src/handler/create_sql_function.rs

```diff
@@ -205,11 +229,46 @@ pub async fn handle_create_sql_function(
                         .into());
                     }
                 }
-                Err(e) => return Err(ErrorCode::InvalidInputSyntax(format!(
-                    "failed to conduct semantic check, please see if you are calling non-existence functions or parameters\ndetailed error message: {}",
-                    e.as_report()
-                ))
-                .into()),
+                Err(e) => {
+                    if let ErrorCode::BindErrorRoot { expr: _, error } = e.inner() {
```

Comment:
Could you extract this block into a function? Adding hint for a SQL statement seems to be a general logic that can be reused by other commands. 🤔